test/1_stdlib/UnsafePointer.swift.gyb

```diff
@@ -322,6 +336,33 @@ UnsafeMutablePointerTestSuite.test("initializeFrom.Right") {
   }
 }
 
+UnsafeMutablePointerTestSuite.test("initializeFrom/immutable") {  
+  var ptr = UnsafeMutablePointer<Missile>(allocatingCapacity: 3)
```

Comment:
Let's not leak memory in tests -- this way memory debugging tools will help us more:

```
defer {
  ptr.deinitialize(count: 3)
  ptr.deallocateCapacity(3)
}
```